lib/exercises.js

```diff
@@ -1,29 +1,74 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: o(n)
+// Space Complexity: o(n)
 function grouped_anagrams(strings) {
```

Comment:
Notice that you are sorting here.  So assuming all the words are limited in length, your time complexity is ok.  If the words can be any length, this is O(n * m log m) where m is the length of the individual words.